app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_memory = {}
+
+    def wrapper(*args) -> dict:
```

Comment:
Change your return type annotation for `wrapper`, it could return `Any` because the function output is not known in advance.